tests/boardfarm_plugins/boardfarm_prplmesh/testsuites.cfg

```diff
@@ -6,3 +6,4 @@
 [test_flows]
 InitialApConfig
 ApConfigRenew
+ClientAssociationLinkMetrics
```

Comment:
This should have been in the first commit.

Reply:
I guess he added it last because test is only working once last commit is in place.